cli/package.json

```diff
@@ -1 +1,35 @@
-{"version": "6.7.0"}
\ No newline at end of file
+{
+  "name": "protobufjs-cli",
+  "description": "Translates between file formats and generates static code as well as TypeScript definitions.",
+  "version": "7.0.0",
+  "author": "Daniel Wirtz <dcode+protobufjs@dcode.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dcodeIO/protobuf.js.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "bin": {
+    "pbjs": "bin/pbjs",
+    "pbts": "bin/pbts"
+  },
+  "scripts": {
+    "prepublish": "node scripts/prepublish"
+  },
+  "peerDependencies": {
+    "protobufjs": "~6.7.0"
```

Comment:
`"^7.0.0"`? (just to avoid having two sets of `pbjs` / `pbts` from different packages)

Reply:
I've updated this to ^7.0.0